oceannavigator/frontend/src/remote/OceanNavigator.js

```diff
@@ -2,21 +2,22 @@
 // that use IE which doesn't support that.
 
 const axios = require("axios");
+import { cacheAdapterEnhancer } from "axios-extensions";
 
-function _createPromise() {
-  return axios.create();
-}
-
+const instance = axios.create({
+  headers: { "Cache-Control": "no-cache" },
+  adapter: cacheAdapterEnhancer(axios.defaults.adapter)
+});
 
 export function GetDatasetsPromise() {
-  return _createPromise().get(
-    "/api/v1.0/datasets"
+  return instance.get(
+    "/api/v1.0/datasets/"
```

Comment:
append `/` to the end of these URLs to avoid a 301 redirect which always busts the cache